recipe/meta.yaml

```diff
@@ -39,6 +39,10 @@ test:
     - pip
   commands:
     - pip check
+    # Check if yajl library is available
+    - if not exist %PREFIX%\\Library\\lib\\yajl_s.lib exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\yajl.dll exit 1  # [win]
+    - python -c "import ijson; print(ijson.backend)"
```

Comment:
It looks like conda chains these commands into a single script, and we exit early if we can't find the yajl dll

Reply:
I removed it